src/TodoList.js

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+import { useRef } from 'react';
+import styled from 'styled-components';
+import TodoListItem from './TodoListItem';
+import InputBar from './InputBar';
+
+const TodoList = () => {
+  const initialData = JSON.parse(localStorage.getItem('todos')) || [];
+  const [data, setData] = useState(initialData);
+  const [doneData, setDoneData] = useState(
+    initialData.filter((item) => item.checked)
+  );
```

Comment:
doneData를 따로 state가 아닌 일반 변수로 관리해도 괜찮을 것 같아요! 아래와 같이 관리하면 setState를 통해 상태변화를 따로 관리하지 않아도 될 것 같아요~~
```suggestion
  const doneDateLength = data.filter((item) => item.checked);
```